roles/lib_openshift/src/test/unit/oc_secret.py

```diff
@@ -81,7 +81,7 @@ def test_adding_a_secret(self, mock_openshift_cmd):
 
         # Making sure our mock was called as we expected
         mock_openshift_cmd.assert_has_calls([
-            mock.call(['get', 'secrets', '-o', 'json', 'secretname'], output=True),
+            mock.call(['get', 'secrets', 'secretname', '-o', 'json'], output=True),
```

Comment:
As long as it's not an unintentional regression on recent changes I'm fine with it.